examples/durable/FunctionChainingApp/requirements.psd1

```diff
@@ -0,0 +1,6 @@
+# This file enables modules to be automatically managed by the Functions service.
+# Only the Azure Az module is supported in preview.
+# See https://aka.ms/functionsmanageddependency for additional information.
+#
+@{
+}
```

Comment:
Please add a line ending. Also, should we add the content of the file commented out? e.g.,
```
# This file enables modules to be automatically managed by the Functions service.
# See https://aka.ms/functionsmanageddependency for additional information.
#
@{
    # For latest supported version, go to 'https://www.powershellgallery.com/packages/Az'.
    # 'Az' = '2.*'
}
```

Reply:
Added line endings. On the commented out content - I don't know why we need that here. I'm trying to make these samples focused specifically on Durable, so these comments look irrelevant and distracting. Perhaps it would even be better to completely remove these requirements.psd1 files...

Reply:
Actually, I agree with you. The example should be as simple as possible and we should not include anything outside Durable.

Reply:
Remove these files.